cli/cmd/dashboard.go

```diff
@@ -55,6 +68,18 @@ var dashboardCmd = &cobra.Command{
 	},
 }
 
+func isDashboardAvailable(client *http.Client, url string) (bool, error) {
+	req, err := client.Get(url)
```

Comment:
on my system this line hangs. the `url` being passed in is `http://127.0.0.1:8001/api/v1/namespaces/conduit/services/web:http/proxy/`, which i don't think will work until `kubernetesProxy.Run()` has been called.

i'm realizing there may be a very quick way to check if conduit is installed, we can simply create a public API client and call `SelfCheck`:

```golang
client, err := newPublicAPIClient()
if err != nil {
	return false, err
}

res, err := client.SelfCheck(context.Background(), &healthcheckPb.SelfCheckRequest{})
if err != nil {
	return false, err
}
for _, result := range res.GetResults() {
	if result.Status != healthcheckPb.CheckStatus_OK {
		return false, nil
	}
}
```

this is a slightly different test than checking a dashboard URL, but may be a more reliable indicator that conduit is installed.

Reply:
Ah! I finally get it! I was confused about how the SelfCheck() would indicate that the dashboard was available. I completely glossed over the fact that the web dashboard is installed together with the public API. For some reason, I thought just pinging the dashboard URL would be sufficient in checking if the dashboard was installed. That method could have worked but it would have meant repeating code that already creates a client to talk to the services in k8s 🤦‍♂️